examples/interaction-middleware/main.go

```diff
@@ -65,8 +65,8 @@ func slackerInteractive(ctx *slacker.InteractionContext) {
 func LoggingInteractionMiddleware() slacker.InteractionMiddlewareHandler {
 	return func(next slacker.InteractionHandler) slacker.InteractionHandler {
 		return func(ctx *slacker.InteractionContext) {
-			ctx.Logger().Infof(
-				"%s initiated \"%s\" with action \"%v\" in channel %s\n",
+			ctx.Logger().Info(
+				"%s initiated \"%s\" with action \"%v\" in channel %s",
```

Comment:
I don't think this is right, have you tested it? I think this will produce something like:

```
2023/11/06 17:36:46 INFO %s initiated "%s" with action "%v" in channel %s $User.ID=$BlockID $ActionID=$Channel.ID
```

To accomplish what I think you want, you need to wrap the string list in `fmt.Sprintf()` to build our string.

Reply:
Good catch. I was doing a mass text replace.